frontend/packages/ceph-storage-plugin/src/components/object-bucket-claim-page/object-bucket-claim.tsx

```diff
@@ -67,7 +68,14 @@ const OBCTableRow: React.FC<RowFunctionArgs<K8sResourceKind>> = ({ obj }) => {
         {storageClassName ? <ResourceLink kind="StorageClass" name={storageClassName} /> : '-'}
       </TableData>
       <TableData className={tableColumnClasses[5]}>
-        <ResourceKebab actions={menuActions} kind={kind} resource={obj} />
+        <ResourceKebab
+          actions={menuActions}
+          kind={kind}
+          resource={obj}
+          hoverMessage={t(
+            'ceph-storage-plugin~Disabled because the resource is being deleted.',
+          )} /* hoverMessage is only visible when kebab is disabled, i.e., when its associated resource is being deleted */
+        />
```

Comment:
```suggestion
        />
```
Is this really a message we want to pass from here. This is applicable to any resource being terminated.

Reply:
ACK will pass a more specific message to our use case.

Reply:
Added this change.